esphome/components/neopixelbus/light.py

```diff
@@ -116,10 +116,16 @@ def validate_method_pin(value):
     'BIT_BANG': 'NeoEsp32BitBang{}Method',
 }
 
-
 def format_method(config):
     variant = VARIANTS[config[CONF_VARIANT]]
     method = config[CONF_METHOD]
+
+    if config[CONF_INVERT] == True:
+        if method == 'ESP8266_DMA':
+            variant = 'Inverted'+variant;
```

Comment:
See also lint checks, the code here should be reformatted

```suggestion
            variant = 'Inverted' + variant
```